datahub-classify/src/datahub_classify/infotype_utils.py

```diff
@@ -91,6 +95,15 @@ def perform_basic_checks(
         config_dict[PREDICTION_FACTORS_AND_WEIGHTS].get(VALUES, None)
         and len(values) < minimum_values_threshold
     ):
+        logger.warning(
+            f"The number of values for column {metadata.name}"
+            f"does not meet minimum threshold for {infotype}"
+        )
+        basic_checks_status = False
+    elif config_dict[EXCLUDE_NAME] is not None and metadata.name in config_dict.get(
```

Comment:
I feel let's preprocess the metadata.name and exclude name list (for e.g. remove special characters, use lowercase letters, etc.) before performing a exact match. It might help in generalizing the solution.
E.g. Cases like metadata.name="Email Sent"  and ExcludeName=["email_sent"] will find a match.

Reply:
Hmm not sure if they'd want this or not, but it's useful enough to put behind a configurable flag I think. I'll do that